package.json

```diff
@@ -31,6 +31,7 @@
         "@definitelytyped/utils": "latest",
         "@octokit/core": "^3.5.1",
         "@octokit/rest": "^16.0.0",
+        "comment-json": "^4.2.3",
```

Comment:
IMO this dependency can be avoided IMO when using prettier own config lookup methods, see other comment

Reply:
Ah, but `tslint.json` files can have comments too.

Reply:
oh, you're absolutely correct (no surprise here regarding topic, btw). Thanks!